networks/suzuka/suzuka-full-node/src/partial.rs

```diff
@@ -24,11 +24,36 @@ use std::future::Future;
 use std::sync::Arc;
 use std::time::Duration;
 
+/// Suzuka partial node has the ability to
 pub struct SuzukaPartialNode<T> {
+	/// The executor responsible for executing transactions and blocks.
+	///
+	/// 'T' must implement the `SuzukaExecutor` trait, allowing different
+	/// types of executors to be used with the `SuzukaPartialNode`.
```

Comment:
Private members do not really need doc comments, as they won't be rendered in the API documents. A plain one-line comment describing the purpose should be enough.

Reply:
There are tonnes of private members in aptos core that are doc commented. This is for IDE support more than rust crate docs I suppose, for eg.

```rust
/// See [AptosDB::get_transaction_by_version].
        ///
        /// [AptosDB::get_transaction_by_version]: ../aptosdb/struct.AptosDB.html#method.get_transaction_by_version
        fn get_transaction_by_version(
            &self,
            version: Version,
            ledger_version: Version,
            fetch_events: bool,
        ) -> Result<TransactionWithProof>;
        
        ```

Reply:
I just checked and you are right, this gives better quality of life in the IDE.